examples/transforms/global_vs_local_translation.rs

```diff
@@ -0,0 +1,116 @@
+use bevy::prelude::*;
+
+struct SomeEntity {
```

Comment:
I would prefer another name for this struct but I don't have any good idea for now...

Reply:
I'm going with EntityState for now since I use this struct mainly for state information.